.travis.yml

```diff
@@ -2,6 +2,9 @@ language: ruby
 rvm:
   - 2.3.0
 script:
+  - bin/rake db:create RAILS_ENV=test
   - bin/rake db:migrate RAILS_ENV=test
   - bundle exec rspec --color --format documentation
   - bundle exec rubocop
+before_script:
+  - cp config/database.yml.travis config/database.yml
```

Comment:
Yeah I saw that – no idea. If we use the env then we've got a portable solution that'll work anywhere, but tbh lets just go with the travis method and think about it later.